README.md

```diff
@@ -103,7 +108,8 @@ database and to create queries. To create a new `GeoFire` instance you need to a
 reference.
 
 ```java
-GeoFire geoFire = new GeoFire(new Firebase("https://<your-firebase>.firebaseio.com/"));
+DatabaseReference ref = FirebaseDatabase.getInstance().getReference("path/to/geofire");
```

Comment:
I don't think it's necessary, we do link to the setup guide earlier. After the setup guide is done this line of code should work.